Classes/Notifications/NotificationSectionController.swift

```diff
@@ -15,7 +15,7 @@ SwipeCollectionViewCellDelegate {
 
     private let client: NotificationClient
     private let dataSource: NotificationsDataSource
-
+    private static var selectedCellId : String? = ""
```

Comment:
Why did you opt for a static variable here?

Reply:
Ya, let's make this a local variable. Also can it just be a `Bool` since there's only one cell?

Reply:
if i declared as local variable to NotificationSectionController then  i seen NotificationsViewController returning  ex- 5 cell then 5 NotificationSectionController and all are having selectedCellId to default as false if i use .  and your correct since there's only one cell per NotificationSectionController storing the state change to how can know with Bool so that i can rest to false . so **i choose global to all section Controller** to track state change  .??? and in deinit in releasing the variable form memory..

Reply:
@TheCodeTalker sorry having trouble following. There is only one `NotificationSectionController` instance per cell, so tracking a single `Bool` per section controller will map directly to the cell. Global state will inevitably lead to trouble.

Reply:
@rnystrom   is the click of cell changes how to reset the bool value in that case???